### ISSUE TITLE
Run generate [traefik] certificates pipeline automatically

### ISSUE THREAD

**Issue by YuryHrytsuk** [ITISFoundation/osparc-ops-environments#608]
Right now the certificate update is done by manually triggering certificates pipeline. In case, we forget, the pipelines will get expired. this should be automated.

We are a bit safe because we have certificates test that will warn us 1 month ahead of expiration.

**Comment by YuryHrytsuk** [comment]
It is not straightforward with STAG and PROD environments. On this environments we run only tagged code which poses a question.

**Comment by YuryHrytsuk** [comment]
Blocked by https://github.com/ITISFoundation/osparc-ops-environments/issues/654

**Comment by YuryHrytsuk** [comment]
UPD 2:
I don't see this step to bring much value. We can achieve the same with existing functionality: we can trigger manually SSL CERT Pipeline during every release. So, it is now worth working until we have properly zero-downtime restartable traefik. 

Blocked until #654 is done

**Comment by mrnicegyu11** [comment]
@YuryHrytsuk is free to decide on the follo-up, either a `wontfix` and we keep doing it manually, or we conduct some test (locust, etc.) to probe traefik downtimes etc. on zmt-master

**Comment by YuryHrytsuk** [comment]
This will be not needed once we migrate to Kubernetes and `cert-manager`

**Comment by YuryHrytsuk** [comment]
WON'T DO. It is done by `cert-manager` on Kubernetes
